src/busctl/busctl.c

```diff
@@ -2543,25 +2527,13 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
 
                 case 'j':
-                        if (on_tty())
-                                arg_json = JSON_PRETTY;
-                        else
-                                arg_json = JSON_SHORT;
+                        arg_json_format_flags = JSON_FORMAT_PRETTY_AUTO|JSON_FORMAT_COLOR_AUTO;
```

Comment:
Is JSON_SHORT the equivalent of JSON_FORMAT_NEWLINE? Will the output be the same?


```
if (((flags & (JSON_FORMAT_PRETTY_AUTO|JSON_FORMAT_PRETTY)) == JSON_FORMAT_PRETTY_AUTO))
    flags |= on_tty() ? JSON_FORMAT_PRETTY : JSON_FORMAT_NEWLINE;
```

Reply:
The old `JSON_SHORT` was equivalent to `JSON_FORMAT_NEWLINE|JSON_FORMAT_COLOR_AUTO`, hence yes, the output should not change by this. (see the json_dump_with_flags() function that is removed further up fo the translation of JSON_SHORT to the flags.